features/XouBaseApiLoginAndRegister/Service/Urls.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Urls.swift
+//  XouDevSpec
+//
+//  Created by Xander Schoeman on 2020/04/20.
+//
+
+import Foundation
+
+public struct UrlStrings {
+    public let getUsersUrl = "https://xoubase.herokuapp.com/users"
+    public let getUsersLocalUrl = "http://localhost:8080"
+
+    public init() {
+    }
```

Comment:
same here this is not required.

Reply:
It also gives me a internal protection error if I do not include it xD. pods are weird like that.If there is a way to bypass it do tell... xD